File: web3rpc/rpc-specs/code-samples/javascript/debug/others/setMutexProfileFraction.js
const { JsonRpcProvider } = require("@kaiachain/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-kairos.node.kaia.io");
  const rate = 2;

  provider.debug
    .setMutexProfileFraction(rate, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();

File: web3rpc/rpc-specs/code-samples/python/debug/others/setMutexProfileFraction.py
from web3 import Web3
from web3py_ext import extend

host = "https://public-en-kairos.node.kaia.io"

rate = 2

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.set_mutex_profile_fraction(rate)

print(debug_response)
